[2023-08-22 14:26:54,113] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: new_etl.transformar_data manual__2023-08-22T14:17:41.635426+00:00 [queued]>
[2023-08-22 14:26:54,120] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: new_etl.transformar_data manual__2023-08-22T14:17:41.635426+00:00 [queued]>
[2023-08-22 14:26:54,121] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-08-22 14:26:54,121] {taskinstance.py:1377} INFO - Starting attempt 1 of 4
[2023-08-22 14:26:54,121] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-08-22 14:26:54,130] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): transformar_data> on 2023-08-22 14:17:41.635426+00:00
[2023-08-22 14:26:54,138] {standard_task_runner.py:52} INFO - Started process 6541 to run task
[2023-08-22 14:26:54,142] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'new_etl', 'transformar_data', 'manual__2023-08-22T14:17:41.635426+00:00', '--job-id', '161', '--raw', '--subdir', 'DAGS_FOLDER/etl2.py', '--cfg-path', '/tmp/tmpwysfz3rp', '--error-file', '/tmp/tmpxi3j49x2']
[2023-08-22 14:26:54,143] {standard_task_runner.py:80} INFO - Job 161: Subtask transformar_data
[2023-08-22 14:26:54,218] {task_command.py:371} INFO - Running <TaskInstance: new_etl.transformar_data manual__2023-08-22T14:17:41.635426+00:00 [running]> on host 0110d0e687e9
[2023-08-22 14:26:54,327] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Sol
AIRFLOW_CTX_DAG_ID=new_etl
AIRFLOW_CTX_TASK_ID=transformar_data
AIRFLOW_CTX_EXECUTION_DATE=2023-08-22T14:17:41.635426+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-22T14:17:41.635426+00:00
[2023-08-22 14:26:54,367] {python.py:173} INFO - Done. Returned value was: None
[2023-08-22 14:26:54,381] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=new_etl, task_id=transformar_data, execution_date=20230822T141741, start_date=20230822T142654, end_date=20230822T142654
[2023-08-22 14:26:54,439] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-22 14:26:54,473] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
